org.adempiere.base/src/org/compiere/model/MInOut.java

```diff
@@ -991,14 +977,40 @@ public void setM_Warehouse_ID (int M_Warehouse_ID)
 		}
 	}	//	setM_Warehouse_ID
 
+	public static String getMovementType(Properties ctx, int C_DocType_ID, boolean issotrx, String trxName) {
+		String movementType = null;
+		MDocType docType = new MDocType(ctx, C_DocType_ID, trxName);
```

Comment:
use MDocType docType = MDocType.get(C_DocType_ID) instead. And to check C_DocType_ID is valid, we can add the checking if (docType == null) return null; (or just throw illegalargumentexception).